### PR TITLE
Add few new SSD Drives and remove ADATA SU650

### PR DESCRIPTION
**Pull request by PVasileff**
Added "Samsung PM871b" , "Samsung SSD 860 EVO 250GB" and "Silicon Motion based SSDs" Removed ADATA SU650, because missing correct names of Smart ATTRIBUTES and it is  Silicon Motion based SSDs

### CODE REVIEW

File: check_smartdb.json [hunk 1]
```diff
@@ -26,14 +26,14 @@
 				"0" : ["0","0"],
 				"1" : ["50","60"],
 				"2" : ["16:","11:"],
-				"4" : ["85","90"],
+				"4" : ["70","80"],
```
Comment (gschoenberger) [inline check_smartdb.json:29]: Why did you change the default settings of the generic NVMe device @PVasileff ?

Comment (PVasileff) [reply check_smartdb.json:29]: Sorry about that. I have copied whole my file. You can close that. I'll make a new push request without changes of default settings.

Comment (PVasileff) [reply check_smartdb.json:29]: New pull request: https://github.com/thomas-krenn/check_smart_attributes/pull/118
with default parameters. FYI,  аbout the temperature values - some manufacturers of NVME drives have lower temperature thresholds than you set.
